vulcanai/models/utils.py

```diff
@@ -33,17 +33,16 @@ def get_probs(network, loader, index_to_iter, ls_feat_vals):
     for index in range(len(loader)):
         dct_scores[index] = {}
     for index in range(len(loader)):
-        nn_subjFeat = torch.Tensor(loader.dataset[index][0].numpy())
         nn_subjLabel = torch.LongTensor(loader.dataset[index][1].float().numpy())
```

Comment:
weeeeelll may as well be consistent? why keep one as a variable and not the other? Also now we have one that's converted to numpy and one that maybe appears not to be?

Reply:
nevermind, noticed it's outdated.